### PR TITLE
Add example to DictType

### PR DESCRIPTION
**Pull request by charlax**
It is unclear if `type` is (1) a Python `type` (2) a schematics type defined in `schematics.types`. Adding an example clarifies which is which to avoid exceptions such as `AttributeError: 'str' object has no attribute 'owner_model'`.

### CODE REVIEW

**Comment by kracekumar** [comment]
@charlax Thanks. `ListType` also needs an example, can you add ?

File: schematics/types/compound.py [hunk 1]
```diff
@@ -244,6 +244,13 @@ def _export(self, list_instance, format, context):
 class DictType(CompoundType):
     """A field for storing a mapping of items, the values of which must conform to the type
     specified by the ``field`` parameter.
+    
+    Use it like this:
+    
+    .. code-block:: python
```
Comment (bintoro) [inline schematics/types/compound.py]: The `code-block` directive is unnecessary. A simple `::` marker is preferred (`Use it like this::`).

Comment (charlax) [inline schematics/types/compound.py]: Yes - but only if Python is the default highlighted language. I usually prefer to be explicit. Let me know if you feel strongly about this and I'll make the change.

**Comment by charlax** [comment]
Added the example.

**Comment by bintoro** [comment]
Thanks! Merged via 0ead9c19a9e48de13d7d3967c3e9781fac2f45c0.

**Pull request closed without merging**